src/main/java/com/google/javascript/gents/RemoveGoogScopePass.java

```diff
@@ -0,0 +1,69 @@
+package com.google.javascript.gents;
+
+import com.google.javascript.jscomp.AbstractCompiler;
+import com.google.javascript.jscomp.CompilerPass;
+import com.google.javascript.jscomp.NodeTraversal;
+import com.google.javascript.rhino.Node;
+import com.google.javascript.rhino.Token;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Compiler pass removing the {@code goog.scope} wrapper, so that the module and class
+ * transformations can take effect.
```

Comment:
Good point.

Unfortunately, this breaks the "aggressive" exporting.  I'd like to leave this as a TODO for a future cl, if possible.